src/js/cookieparser.js

```diff
@@ -0,0 +1,21 @@
+// Module to add body parsing to express.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.middleware.cookieparser");
+
+var cp = require("cookie-parser");
+
+gpii.express.middleware.cookieparser.getCookieParserMiddlewareFunction = function () {
```

Comment:
I think we can shed "Function" from this name given a piece of middleware is already necessarily a function.